src/test/suite/fixtures/recorded/selectionTypes/postColumnWhaleUntilEach.yml

```diff
@@ -0,0 +1,53 @@
+spokenForm: post column whale until each
+languageId: markdown
+command:
+  actionName: setSelectionAfter
+  partialTargets:
+    - type: range
+      start:
+        type: primitive
+        selectionType: column
+        mark: {type: decoratedSymbol, symbolColor: default, character: w}
+      end:
+        type: primitive
+        mark: {type: decoratedSymbol, symbolColor: default, character: e}
+      excludeStart: false
+      excludeEnd: true
+  extraArgs: []
+initialState:
+  documentContents: |
+    hello world !
+    hi there
+    Second things to say
+    and I'm here
+  selections:
+    - anchor: {line: 0, character: 6}
+      active: {line: 0, character: 6}
+  marks:
+    default.w:
+      start: {line: 0, character: 6}
+      end: {line: 0, character: 11}
+    default.e:
+      start: {line: 3, character: 8}
+      end: {line: 3, character: 12}
+finalState:
+  documentContents: |
+    hello world !
+    hi there
+    Second things to say
+    and I'm here
+  selections:
+    - anchor: {line: 0, character: 11}
+      active: {line: 0, character: 11}
+    - anchor: {line: 1, character: 8}
+      active: {line: 1, character: 8}
```

Comment:
Oh man this really confused me for a while 😅.  We should figure out some way to add comments to our recorded tests.  I see now that this is testing that it clamps to line end; nice test

Reply:
Yeah some test becomes quite esoteric. But you probably had reason for recording it in the first place :D